### PR TITLE
feat: onboard WXP samples

### PR DESCRIPTION
**Pull request by tecton**
1. Add Capability filter option: Word, Excel.
image
2. Onboard 6 samples:
#1214
#1216
#1217
#1218
#1219
#1224
image

### CODE REVIEW

File: .config/samples-config-v3.json [hunk 1]
```diff
@@ -789,6 +790,175 @@
...
+        },
+        {
+            "id": "Word-Add-In-Writing-Assistant",
+            "shortId": "word-writing-assistant",
+            "onboardDate": "2024-03-29",
+            "title": "Word Add-in Writing Assistant",
+            "shortDescription": "This add-in demonstrates Word add-in capabilities to help users check errors, improve writing, and rephrase content.",
+            "fullDescription": "This add-in demonstrates Word add-in capabilities to help users check errors, improve writing, and rephrase content.",
+            "types": [
+              "Office Add-in"
```
Comment (hermanwenhe) [inline .config/samples-config-v3.json:802]: Word

File: .config/samples-config-v3.json [hunk 2]
```diff
@@ -789,6 +790,175 @@
...
+        },
+        {
+            "id": "Excel-Add-in-ChartAPI-Anylysis-Data",
+            "shortId": "excel-chartapi-analysis-data",
+            "onboardDate": "2024-03-29",
+            "title": "Using Chart API to Analyse Data",
+            "shortDescription": "The sample add-in demonstrates Excel add-in capablities to help users using chart API to analyse data.",
+            "fullDescription": "The sample add-in demonstrates Excel add-in capablities to help users using chart API to analyse data.",
+            "types": [
+              "Office Add-in"
```
Comment (hermanwenhe) [inline .config/samples-config-v3.json:830]: Excel

File: .config/samples-config-v3.json [hunk 3]
```diff
@@ -789,6 +790,175 @@
...
+        },
+        {
+            "id": "Excel-Add-in-ShapeAPI-Dashboard",
+            "shortId": "excel-shapeapi-dashboard",
+            "onboardDate": "2024-03-29",
+            "title": "Using Shape API to Work as a Dashboard",
+            "shortDescription": "The sample add-in demonstrates Excel add-in capablities to help users using shape API to work as a dashboard.",
+            "fullDescription": "The sample add-in demonstrates Excel add-in capablities to help users using shape API to work as a dashboard.",
+            "types": [
+              "Office Add-in"
```
Comment (hermanwenhe) [inline .config/samples-config-v3.json:858]: Excel

File: .config/samples-config-v3.json [hunk 4]
```diff
@@ -789,6 +790,175 @@
...
+        },
+        {
+            "id": "Excel-Add-in-Custom-Function-Using-WebWorker",
+            "shortId": "excel-webworker-custom-function",
+            "onboardDate": "2024-03-29",
+            "title": "Custom Function Sample using Web Worker",
+            "shortDescription": "Shows how to use web workers in custom functions to prevent blocking the UI of your Office Add-in.",
+            "fullDescription": "Shows how to use web workers in custom functions to prevent blocking the UI of your Office Add-in.",
+            "types": [
+              "Office Add-in"
```
Comment (hermanwenhe) [inline .config/samples-config-v3.json:886]: Excel

File: .config/samples-config-v3.json [hunk 5]
```diff
@@ -789,6 +790,175 @@
...
+        },
+        {
+            "id": "Excel-Add-in-Mail-Merge",
+            "shortId": "excel-mail-merge",
+            "onboardDate": "2024-03-29",
+            "title": "Create an Excel Mail Merge Add-in",
+            "shortDescription": "This sample demonstrates how to use the Microsoft Graph JavaScript SDK to send emails in Excel from Office Add-ins.",
+            "fullDescription": "This sample demonstrates how to use the Microsoft Graph JavaScript SDK to send emails in Excel from Office Add-ins.",
+            "types": [
+              "Office Add-in"
```
Comment (hermanwenhe) [inline .config/samples-config-v3.json:915]: Excel

File: .config/samples-config-v3.json [hunk 6]
```diff
@@ -789,6 +790,175 @@
...
+        },
+        {
+            "id": "Word-Add-In-AIGC",
+            "shortId": "word-aigc",
+            "onboardDate": "2024-03-29",
+            "title": "Word Add-in AIGC",
+            "shortDescription": "This sample shows a practical Word Add-in with content insertion features, utilizing either AI or predefined content.",
+            "fullDescription": "This sample shows a practical Word Add-in with content insertion features, utilizing either AI or predefined content.",
+            "types": [
+              "Office Add-in"
```
Comment (hermanwenhe) [inline .config/samples-config-v3.json:944]: Word

File: .config/samples-config-v3.json [hunk 7]
```diff
@@ -4,7 +4,8 @@
             "Tab",
             "Bot",
             "Message Extension",
-            "Meeting"
+            "Meeting",
+            "Office Add-in"
```
Comment (hermanwenhe) [inline .config/samples-config-v3.json:8]: Word, Excel

**Review by summzhan — APPROVED** [review]

**Pull request merged**